build-scripts/generateNonCodeFiles.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+    This script is called from npm run compile.
+    It creates an HTML version of the marketplace page to be used as a quick start page.
```

Comment:
Because the script is responsible for more than one operation, split the operations into separate functions, and put a descriptive function comment on each, so that future maintainers can find and work with the feature they're interested in. Revise the file heading.

Reply:
Split into functions